lib/endpoints/class-wp-rest-attachments-controller.php

```diff
@@ -122,10 +123,11 @@ public function update_item( $request ) {
 			update_post_meta( $data['id'], '_wp_attachment_image_alt', $request['alt_text'] );
 		}
 
-		$response = $this->get_item( array(
-			'id'      => $data['id'],
-			'context' => 'edit',
-		));
+		$attachment_location = rest_url( '/wp/v2/' . $this->get_post_type_base( get_post_type( $data['id'] ) ) . '/' . $data['id'] );
```

Comment:
> Is it necessary that we pass the method and url to WP_REST_Request?

Huh. That's actually a good question. How do you feel about it? Is there harm in including it?

In my opinion, we'd ideally be calling through the server: https://github.com/WP-API/WP-API/issues/1218#issuecomment-103655685 If we call through the server, we'd need the URL on the request object so the server knows how to appropriately route it.